wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -305,7 +305,7 @@ <h1>Error: HTTP {{ status_code }} -- {{ reason }}</h1>
     <div class="user-feedback">
```

Comment:
This thought just occurred to me. I wonder if this should go on all pages. The networks pages are public so wondering if this bit of html should be part of the footer? Let's chat with Shawna about this at stand-up. It will be good to distinguish which button they are clicking.